README.md

```diff
@@ -11,60 +11,70 @@ on IBM Blockchain.  The components of the demo are:
 * A Trade Center for buying and selling existing trades.
 * A special interface just for auditors of the network to examine trades
 
+##### Versions and Supported Platforms
+On November 9th, 2016, we released the IBM Blockchain Service v1.0 based on HyperLedger fabric v0.6.  All new networks created in bluemix will be this version.  Support of the v0.4.2.x Bluemix Service based on the 0.5.3 Hyperledger Fabric has been deprecated.  It is strongly recommended that if you have an existing network based on 0.5.3, you redeploy a new network and follow the instructions in the 2.0 branch.
+
+- [CP-Web - Branch v2.0](https://github.com/ibm-blockchain/cp-web/tree/v2.0)
+	- Works with Hyperledger fabric `v0.6-developer-preview`
+
+If for some reason you need instructions for the v0.4.2+ level of the service, they are here, but support of these instructions is
+best effort only.
+
+- [CP-Web - Branch v1.0](https://github.com/ibm-blockchain/cp-web/tree/v1.0) (Deprecated)
+	- Works with Hyperledger fabric `v0.5-developer-preview`
+	- IBM Bluemix Blockchain Service `v0.4.2+`
+
 ## Getting Started
 
-1. Deploy the demo to your [IBM Bluemix](https://www.bluemix.net/) account using the button above.
+1. Deploy the demo to your [IBM Bluemix](https://www.bluemix.net/) account using this button [![Deploy to Bluemix](https://bluemix.net/deploy/button.png)](https://bluemix.net/deploy?repository=https://github.com/IBM-Blockchain/cp-web.git)  
 
-##### OR
+`OR`  
 
-1. Clone this repository.
-2. Create an instance of the IBM Blockchain service in the Bluemix catalog.
-3. Copy the credentials from the service into the file 'my_creds.json'.
-4. Make sure the key/value store only has values for your current network (See below).
-5. Run these commands in the cloned directory:
+1. Deploy the app on my local machine, connecting to an IBM Blockchain network running in Bluemix  - [instructions](#manbluenetwork)
 
-```shell
-npm install
-gulp
-```
 
-These credentials can be obtained from the "Service Credentials" tab of the Bluemix service. They are
-in the form:
-
-```json
-{
-  "credentials": {
-    "peers": [
-      {
-        "discovery_host": "169.53.62.121",
-        "discovery_port": "40275",
-        "api_host": "169.53.62.121",
-        "api_port": "40276",
-        "type": "peer",
-        "network_id": "4b21f2f9-4d10-4946-a0df-f91ac09dbc03",
-        "id": "4b21f2f9-4d10-4946-a0df-f91ac09dbc03_vp1",
-        "api_url": "http://169.53.62.121:40276"
-      }
-    ],
-    "users": [
-      {
-        "username": "user_type0_b7c7a1e545",
-        "secret": "89ce33e4e6"
-      }
-    ]
-  }
-}
-```
+# <a name="manbluenetwork"></a>Deploy the CP-Web App locally and connect to an IBM Blockchain Network running in Bluemix:
+
+1.  Follow these instructions to [Set up your environment for running the demos](https://github.com/ptippett/marbles/blob/break_out_common_sections/demo_prereqs)
```

Comment:
ok..that was a mistake..should be a .md.  I'll fix.  I do use that link you submitted.  All my intermediate link does is:

tell them to ignore this if they did learn chaincode.
give them your link, but tell them all they need to do is git and node.js.  Not everything else.

reducing that to a single link might be a good idea, just didn't do it.